service/client_state.hh

```diff
@@ -156,7 +156,12 @@ public:
         _driver_version = std::move(driver_version);
     }
 
-    client_state(external_tag, auth::service& auth_service, qos::service_level_controller* sl_controller, timeout_config timeout_config, const socket_address& remote_address = socket_address(), bool thrift = false)
+    client_state(external_tag,
+                 auth::service& auth_service,
+                 qos::service_level_controller* sl_controller,
+                 timeout_config timeout_config,
+                 const socket_address& remote_address = socket_address(),
+                 bool thrift = false)
```

Comment:
nit: I appreciate changes like that, but this particular one is alien to this patch

Reply:
i will piggy back it as a dedicated commit in this patchset.